pkg/smartcontract/parameter_test.go

```diff
@@ -71,7 +73,7 @@ var marshalJSONTestCases = []struct {
 		input: Parameter{
 			Type: MapType,
 			Value: map[Parameter]Parameter{
-				{Type: StringType, Value: "key1"}: {Type: IntegerType, Value: 1},
+				{Type: StringType, Value: "key1"}: {Type: IntegerType, Value: int64(1)},
```

Comment:
Not a part of efa619dfcca4a8cccd01b7a8e4fa9052ac3a0ad3, it's a fix for an earlier commit.